src/docs/sphinx/Actions/Pipelines.rst

```diff
@@ -1089,33 +1089,26 @@ among many simulation ranks. Ascent utilizes the ``partition()`` functions provi
 that can be used to split or recombine Blueprint meshes in serial or parallel.
 Full M:N repartioning is supported. The ``partition()`` functions are in the
 serial and parallel Blueprint libraries, respectively.
+Funtionality and further descriptions of optional parameters can be found in the Conduit::Blueprint `documentation <https://llnl-conduit.readthedocs.io/en/latest/blueprint_mesh_partition.html#partitioning>`_.
 
 .. code:: cpp
 
-    // Serial
-    void conduit::blueprint::mesh::partition(const Node &mesh,
-                                             const Node &options,
-                                             Node &output);
-
-    // Parallel
-    void conduit::blueprint::mpi::mesh::partition(const Node &mesh,
-                                                  const Node &options,
-                                                  Node &output,
-                                                  MPI_Comm comm);
-
-
-Partitioning meshes using Blueprint will use any options present to determine
-how the partitioning process will behave. Typically, a caller would pass options
-containing selections if pieces of domains are desired. The partitioner processes
-any selections and then examines the desired target number of domains and will then
-decide whether domains must be moved among ranks (only in parallel version) and
-then locally combined to achieve the target number of domains. The combining
-process will attempt to preserve the input topology type for the output topology.
-However, in cases where lower topologies cannot be used, the algorithm will promote
-the extracted domain parts towards more general topologies and use the one most
-appropriate to contain the inputs.
-
-In parallel, the ``partition()`` function will make an effort to redistribute data across MPI
+  conduit::Node pipelines;
+  // pipeline 1
+  pipelines["pl1/f1/type"] = "partition";
+  //params optional
+  pipelines["pl1/f1/params/target"] = 1;
+  const char* fields[3] = {"pink", "pony", "club"};
+  pipelines["pl1/f1/params/fields"].set(fields,3);
```

Comment:
I think this will only end up wrapping the first 3 letters of the char *
For a list of strings, they need to be a conduit list, where each entry is a string.